src/lib_archives.mli

```diff
@@ -0,0 +1,13 @@
+open Stdune
```

Comment:
This is about a bit more than just library archives, what about `Lib_installed_files`?

Reply:
Note that it doesn't in fact include all installed files. For example, it doesn't include the sources (anymore) or the ppx driver related stuff. We can also push the module specific stuff away from it and stick to archives only for `@all`. cmi's, cmt's, etc. will all be built transitively anyways.

Reply:
that seems simpler indeed. And thiking about it more, if we really want to reference these files explicitely, we can also depend on all the files in the object directory

Reply:
Okay, I've done that.